.github/workflows/ci.yml

```diff
@@ -138,7 +138,8 @@ jobs:
 
     strategy:
       matrix:
-        test-name: [ 'generic', 'nextjs', 'express', 'quickstart']
+        test-name: [ 'generic', 'nextjs', 'express', 'quickstart', 'ap-flows' ]
+        test-project: [ 'chrome' ]
```

Comment:
❓ we are installing webkit but we don't seem to run the webkit project. Is this intended?

Reply:
Removed from CI